noxfile.py

```diff
@@ -229,6 +227,8 @@ def tests(session: nox.Session, test_specifier: nox._parametrize.Param) -> None:
             session.run_install(
                 *uv_sync,
                 "--extra=tests",
+                "--no-default-extras",
+                f"--python={session.virtualenv.location}",
```

Comment:
These additional lines are from https://github.com/wntrblm/nox/pull/933, which added them to the Nox + uv recipe.